File: checkers/140/main.go
package main

import (
	"fmt"
	"testlib"
	"testlib/core"
)

func main() {
	defer func() {
		_ = testlib.GetStream(testlib.TaskOutput).Close()
		_ = testlib.GetStream(testlib.InputFromUser).Close()
	}()
	// inputStream := testlib.GetReader(testlib.TaskInput)
	outputStream := testlib.GetReader(testlib.TaskOutput)
	inputFromUserStream := testlib.GetReader(testlib.InputFromUser)

	out := core.ReadWord(inputFromUserStream)
	ans := core.ReadWord(outputStream)

	for out[len(out)-1] == 32 {
		out = out[0 : len(out)-1]
	}
	for ans[len(ans)-1] == 32 {
		ans = ans[0 : len(ans)-1]
	}
	if ans == out {
		testlib.WriteAnswer("Full solution", testlib.OK)
	} else {
		testlib.WriteAnswer(fmt.Sprintf("Incorrect user output. User answer: %d. Correct answer: %d", out, ans), testlib.WA)
	}
}

File: checkers/140/main.cpp
#include <checkers/testlib.h>
#include <iostream>
#include <string>
#include <vector>
#include <algorithm>

using namespace NTestlib;

int main(int argc, char *argv[])
{
    InitChecker(argc, argv);
    std::string out = "", answer = "";
    // File - programm input (from .in file)
    // Out - User output
    // Ans - Correct output (from .out file)
    out = Out.ReadLine();
    answer = Ans.ReadLine();
    while (out[out.size() - 1] == 32)
        out = out.substr(0, out.size() - 1);
    while (answer[answer.size() - 1] == 32)
        answer = answer.substr(0, answer.size() - 1);
    if (out == answer)
        QuitWith(AC, "Full solution");
    std::stringstream msg;
    msg << "Out " << out.c_str() << ", Correct answer: " << answer.c_str();
    QuitWith(WA, msg.str());
}